ndcube/ndcube.py

```diff
@@ -258,7 +259,7 @@ def world_to_pixel(self, *quantity_axis_list):
         result = [u.Quantity(world_to_pixel[index], unit=u.pix) for index in range(self.wcs.pixel_n_dim)]
         return result[::-1]
 
-    def axis_world_coords(self, *axes, edges=False):
+    def axis_world_coords(self, axes=(), edges=False, skycoord=True):
```

Comment:
I think `values=False` would be a more consistent way of wording it with the APE 14 API. But then you only handle the SkyCoord case explicitly :thinking:

What's the motivation for this ooi?

Reply:
Umm in the plotting code there are various areas where the axis_world_coords returns a 2d array, but now it returns a skycoord so that makes the thing dicey. So I thought of using a param to return a 2d array of skycoord.